src/main/resources/sessionLinksRecoveryEmailTemplate-emailNotFound.html

```diff
@@ -0,0 +1,20 @@
+<p>Hello,</p>
+
+<p>
+    You are receiving this email because an access link recovery request for feedback sessions related to this email address ${userEmail} was made on <a href="${teammateHomePageLink}">TEAMMATES</a>.
```

Comment:
recovery or resend?
`a request to resend TEAMMATES links` might be easier to understand for the end-user.